virtcontainers/pkg/cloud-hypervisor/Makefile

```diff
@@ -13,7 +13,7 @@ YQ := $(shell command -v yq 2> /dev/null)
 generate-client-code: clean-generated-code
 	docker run --rm \
 		--user $$(id -u):$$(id -g) \
-		-v $${PWD}:/local openapitools/openapi-generator-cli generate \
+		-v $${PWD}:/local openapitools/openapi-generator-cli:v4.3.1 generate \
```

Comment:
/cc @devimc I added the same tag to the `openapi-generator-cli` container as you did in the SGX PR in kata 2.0 (https://github.com/kata-containers/kata-containers/pull/484). No separate back-porting PR is needed for tagging the container. :)

Reply:
thanks @likebreath - I appreciate it :+1: